label_sync/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# TODO: alpine is smaller, but if I compile Go binary locally it won't run there
+# TODO: when I use FROM: "golang" compiled binary works just fine
+# TODO: I don't know what the standard workflow is, how to use this Docker file
+# TODO: should I compile Go binary and copy it to image in Dockerfile, or should I
+# TODO: compile program inside docker container? 
+#FROM alpine:3.6
+FROM golang
+MAINTAINER lukaszgryglicki@o2.pl
+
+# TODO: What this is for - images are supposed to be minimal, I think it is not needed
```

Comment:
This is needed if you're talking to someone over HTTPS, such as `https://api.github.com/` :)